index.js

```diff
@@ -61,3 +61,39 @@ function setDisconnected(isDisconnected) {
   document.getElementById('con-error').style.display =
     isDisconnected ? 'block' : 'none';
 }
+
+
+function loadStyle(src) {
+  return new Promise(function (resolve, reject) {
+    let link = document.createElement('link');
+    link.href = src;
+    link.rel = 'stylesheet';
+
+    link.onload = () => resolve(link);
+    link.onerror = () => reject(new Error(`Style load error for ${src}`));
+
+    document.head.append(link);
+  });
+}
+
+function getUrlParameter(sParam){
+  var sPageURL = window.location.search.substring(1);
+  var sURLVariables = sPageURL.split('&');
+  for (var i = 0; i < sURLVariables.length; i++){
+    var sParameterName = sURLVariables[i].split('=');
+    if (sParameterName[0] == sParam)
+    {
+        return sParameterName[1];
+    }
+  }
+}
```

Comment:
I think with `url.parse` we can simplify this function.

Reply:
Thanks ashwinvis. yes, cumbersome string-handlings are truely unnecessary.

But after a further invesigation. I found index.js works in index.html , which run in browser environment, not node.js. 
Adding more js lib into index.html makes me **hesitated, I want to keep the browser' runtime dependencies as minified as possible**. 

But luckly I found [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams), which can perform the same way and within comprehensive standards. The usage looks like: 
```js
new URLSearchParams(window.location.search).get("theme")
```

I gonna to handle it.

Reply:
Good job!